src/ja/worker/docker.py

```diff
@@ -43,6 +46,8 @@ def worker_uid(self) -> str:
     def add_job(self, job: Job) -> None:
         if job.uid in self._containers_by_job_uid:
             raise ValueError("Job with UID %s already exists." % job.uid)
+        logger.info("adding job: %s" % job.uid)
```

Comment:
I think it would also make sense to print the docker ids, at least on the debug level